src/main/java/seedu/address/logic/commands/AddPolicyCommand.java

```diff
@@ -34,7 +34,7 @@ public class AddPolicyCommand extends Command {
             + "[" + PREFIX_POLICY_EXPIRY_DATE + "EXPIRY_DATE]\n"
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_POLICY_TYPE + "life "
-            + PREFIX_POLICY_PREMIUM_AMOUNT + "400.00 "
+            + PREFIX_POLICY_PREMIUM_AMOUNT + "300 "
```

Comment:
I think we should standardise to either both premium and coverage having decimals or no decimals. I think they can call this an error as well since both are double.

Reply:
okk edited